contribs/gmf/apps/desktop_alt/less/main.less

```diff
@@ -16,4 +17,23 @@ header div span {
   flex-direction: column;
 }
 
+.import {
+  input[type="file"] {
+    display: none;
+  }
+  .tt-menu {
+    width: 100%;
+    padding: 5px;
+    background-color: white;
+    color: black;
+    line-height: 1.6em;
+    .tt-suggestion {
+     &:hover {
+       background-color: @onhover-color;
+     }
+     cursor: pointer;
+    }
+  }
+}
```

Comment:
Shouldn't it be in a file like `contribs/gmf/less/import.less` to be reusable, not copied?